index.js

```diff
@@ -16,19 +16,23 @@ let Form = ({ afterInitField = x => x, validate = functions, ...config }) => {
       field,
       label: x.label || _.startCase(field),
       value: '', // in case no value is provided, avoids controlled vs uncontrolled warning and need for mobx 4
+      cleanValue: x.value || '',
       get errors() {
         return form.errors[field] || []
       },
       get isValid() {
         return !node.errors.length
       },
       get isDirty() {
-        return changed(node.value, x.value)
+        return changed(node.value, node.cleanValue)
       },
       reset() {
         node.value = F.when(_.isUndefined, '')(x.value)
       },
       validate: () => form.validate([field]),
+      declareClean: () => {
+        node.cleanValue = node.value
```

Comment:
you don't actually need a new `cleanValue` prop here - you can just say `x.value` = `node.value`

Reply:
This method could also just be called `clean` as that is a verb in addition to an adjective